### PR TITLE
Fix items in edge conf

### PR DESCRIPTION
**Pull request by edisonxiang**
**What type of PR is this?**
/kind test

**What this PR does / why we need it**:
* add new items in the edge conf to pass the perf test in current version.
* update edge version.

### CODE REVIEW

File: tests/performance/assets/02-edgeconfigmap.yaml [hunk 1]
```diff
@@ -54,6 +55,14 @@ data:
         docker-address: tcp://localhost:2375
         version: 2.0.0
```
Comment (sujithsimon22) [inline tests/performance/assets/02-edgeconfigmap.yaml]: @edisonxiang should we maybe update the version field as well to match the current version?

Comment (edisonxiang) [reply tests/performance/assets/02-edgeconfigmap.yaml]: Thanks. done

**Comment by kevin-wangzefeng** [comment]
/approve
Thanks.

**Comment by sids-b** [comment]
@edisonxiang , can we make the same change in edgesite.yaml as well ?

**Comment by sids-b** [comment]
https://github.com/kubeedge/kubeedge/blob/master/edgesite/conf/edgeSite.yaml this is the link to the file

**Comment by edisonxiang** [comment]
Thanks @sids-b 
We can change edgeSite.yaml in another pr which is related some other modifications about conf.

**Pull request merged**